src/bin/main.ts

```diff
@@ -1,63 +1,230 @@
 #!/usr/bin/env node
 /**
  * @license GPL3
- * @author RedCrafter07 (https://github.com/RedCrafter07)
+ * @author RedCrafter07 <https://github.com/RedCrafter07>
+ * @author FishingHacks <https://github.com/FishingHacks>
  */
 
 import chalk from 'chalk';
-import { existsSync, lstatSync } from "fs";
-import { readdir } from 'fs/promises';
+import { existsSync, lstatSync, readdirSync, readFileSync } from 'fs';
```

Comment:
Why do you need readdirSync and readFileSync? readdir & readfile from fs/promises to the same.

Reply:
Well, i can change it in the next pr, have a system half-ready and don't want to remove it

Reply:
Okay